x-pack/solutions/security/plugins/security_solution/public/common/components/header_page/index.tsx

```diff
@@ -81,14 +62,30 @@ export interface HeaderPageProps extends HeaderProps {
   titleNode?: React.ReactElement;
 }
 
+const useHeaderLinkBackStyles = () => {
+  const euiThemeContext = useEuiTheme();
+  const fontSizeXs = euiFontSize(euiThemeContext, 'xs').fontSize;
+  const lineHeightS = euiFontSize(euiThemeContext, 's').lineHeight;
```

Comment:
sorry for the probably dumb question, but why using `euiFontSize(euiThemeContext, 'xs').fontSize;` here instead of just `euiThemeContext.font.scale.xs`?

I'm not finding an equivalent for `lineHeight` interestingly. I see `euiThemeContext.font.lineHeightMultiplier` but it wouldn't work for us here...